cli/azd/internal/appdetect/maven_command.go

```diff
@@ -55,12 +54,16 @@ func getMvnwCommand(path string) (string, error) {
 	return "", fmt.Errorf("failed to find mvnw command in project")
 }
 
-const mavenVersion = "3.9.9"
-const mavenZipFileName = "apache-maven-" + mavenVersion + "-bin.zip"
-const mavenURL = "https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/" +
-	mavenVersion + "/" + mavenZipFileName
```

Comment:
Originally, the `const` is used to solve this failure:
https://github.com/azure-javaee/azure-dev/actions/runs/12424133693/job/34688801919

![image](https://github.com/user-attachments/assets/9166fe1e-1ea2-4f51-ab33-b85f14ee407a)

Not this problem is solved by another way in download_util.go:
```
client := &http.Client{
Timeout: 30 * time.Second,
  Transport: &http.Transport{
    Proxy: http.ProxyFromEnvironment,
  },
}
```